modules/ref/pages/client-settings.adoc

```diff
@@ -159,6 +159,16 @@ The Key/Value default timeout is used on operations which are performed on a spe
 key if not overridden by a custom timeout. This includes all commands like get(),
 getFromReplica() and all mutation commands, but does not include operations that are
 performed with enhanced durability requirements.
++
+TIP: xref:concept-docs:durability-replication-failure-considerations.adoc#synchronous-writes[Durable Write operations] have their own timeout setting, `KVDurabilityTimeout`, see below.
+
+Name: *KVDurabilityTimeout*::
```

Comment:
This named KVDurableTimeout in the SDK

Reply:
Indeed - think I got the different name via the ticket :-/
fixed :+1: